src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/RewriteSpec.scala

```diff
@@ -155,6 +155,11 @@ class RewriteSpec extends AnyFlatSpec with XmlSupport with SchemaSupport with Ma
           href="https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:48786">
           Plangebied Harinxmaland, gemeente Sneek. Archeologisch vooronderzoek: een inventariserend veldonderzoek (waarderend onderzoek)
         </ddm:references>
+        <ddm:references
+          href="https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:56023">
+        </ddm:references>
+        <ddm:references>https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:48515</ddm:references>
```

Comment:
```suggestion
        <dc:relation>https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:48515</dc:relation>
```
Should work also. what would be the result?
```
        <ddm:relation href="scheme="id-type:DOI"
            href="https://doi.org/10.17026/dans-xc4-vj4h">
            https://doi.org/10.17026/dans-xc4-vj4h</ddm:relation>
```